std/http.ln

```diff
@@ -61,6 +61,46 @@ export fn fetch(req: Request): Result<Response> {
   }
 }
 
+export fn fetch(req: Request): Result<Response> {
```

Comment:
not very DRY yet, but soon!

Reply:
![soon](http://vignette4.wikia.nocookie.net/safari-zone/images/c/c1/Soon-horse.jpg/revision/latest?cb=20130627171445)